test/lib.list.js

```diff
@@ -176,7 +176,7 @@ describe('List method', () => {
         app.screenshots.map(assertValidUrl);
         app.comments.map(assert.isString);
       }));
-  }).timeout(timeout);
+  }).timeout(timeout * 2);
```

Comment:
this is suspicious, it's probably better to increase the global timeout in the package.json than in the code.

Reply:
suspicious in which  way? The timeout is already defined in the top of the method and only those tests using the fullDetail are having problems with the timeout.
But maybe the change in how to get the full detailed apps resolves this anyway.